packages/taquito/test/estimate/rpc-estimate-provider.spec.ts

```diff
@@ -1588,4 +1588,101 @@ describe('RPCEstimateProvider test wallet', () => {
       done();
     });
   });
+
+  describe('stake', () => {
+    it('return the correct estimate for stake', async (done) => {
+      mockRpcClient.simulateOperation.mockResolvedValue({
+        contents: [
+          {
+            kind: 'transaction',
+            fee: 10017,
+            metadata: {
+              operation_result: { status: 'applied', consumed_milligas: '10013000' },
+            },
+            parameters: {
+              entrypoint: 'stake',
+              value: {
+                prim: 'Unit',
+              },
+            },
+          },
+        ],
+      });
+      // Simulate real op size
+      mockForger.forge.mockResolvedValue(new Array(149).fill('aa').join(''));
+      const estimate = await estimateProvider.stake({
+        amount: 10000,
+      });
+      expect(estimate).toMatchObject({
+        gasLimit: 10113,
+        storageLimit: 0,
+        suggestedFeeMutez: 1361,
+      });
+      done();
+    });
+  });
+
+  describe('unstake', () => {
+    it('return the correct estimate for unstake', async (done) => {
+      mockRpcClient.simulateOperation.mockResolvedValue({
+        contents: [
+          {
+            kind: 'transaction',
+            fee: 10018,
+            metadata: {
+              operation_result: { status: 'applied', consumed_milligas: '10014000' },
+            },
+            parameters: {
+              entrypoint: 'unstake',
+              value: {
+                prim: 'Unit',
```

Comment:
thought the unstake will have `value: { int: '10000000000'}`

Reply:
actually doesn't even need parameters at all for all three op, will be nice to have to if having might want to be correct

Reply:
> thought the unstake will have value: { int: '10000000000'}

did we ever get to confirm with the protocol team whether this value is arbitrary, fixed, or if it has any significance?

Reply:
From the types in Taquito is seems to be required when `entrypoint` is set